tools/bash.py

```
@@ -1,257 +1,230 @@
-    Ensure that the python code handles errors in a way that will allow the execution of the main program to continue.
-    Avoid unnecessary complexity; keep the script concise and readable.
-    If the Bash command is invalid or unsupported, return an error message explaining why.
-    Input: {bash_command}
-    Output: """
-    # rr("Prompt")
+    project_dir = Path(get_constant("PROJECT_DIR"))
+    with open(file_path, "r") as file:
+        prompt_string = file.read()
+    prompt_string += f"Your project directory is {project_dir}. You need to make sure that all files you create and work you do is done in that directory. \n"
+    prompt_string += f"Your bash command is: {bash_command}\n"
+    temp= input(f" The prompt is: {prompt_string}")
```

Ruff (0.8.2)

Local variable `temp` is assigned to but never used

Remove assignment to unused variable `temp`

(F841)